deploy/osd-rebalance-infra-nodes/07-osd-rebalance-infra-nodes.ConfigMap.yaml

```diff
@@ -67,7 +70,7 @@ data:
         LABEL=$3
         for POD in $( oc get pods -n $NS -l $LABEL=$APP -o jsonpath='{.items[*].metadata.name}' ) ; do
             echo "INFO: Waiting for $POD to be Running..."
-            while [ "$(oc get pod -n $NS $POD -o jsonpath='{.status.phase}' 2>/dev/null)" != "Running" ];
+            while [ "$(oc get pod -n $NS $POD -o jsonpath='{.status.phase}' 2>/dev/null)" == "Pending" ];
```

Comment:
This change might alter the behavior to now wait for a phase other than Running (like Terminating, etc.).

Reply:
Thank you for review and pointing it out. I have updated.